src/js/jsx/sections/transform/Position.jsx

```diff
@@ -52,35 +53,37 @@ define(function (require, exports, module) {
                 return collection.pluckAll(layers, ["kind", "locked", "isBackground"]);
             };
 
-            return !Immutable.is(getSelectedChildBounds(this.props), getSelectedChildBounds(nextProps)) ||
+            return this.props.referencePoint !== nextProps.referencePoint ||
```

Comment:
Can you define `getDefaultProps` and `propTypes` (grep the source for examples) to both provide a default reference point and document its type (a string, required)?